### ISSUE TITLE
Change proxy URL 

### ISSUE THREAD

**Issue by prathamesh1997** [anil-sidhu/react-google-location#7]
Can you elaborate on proxy url, and how can I replace it with my own

**Comment by SilverioReyes** [comment]
Yes, I' am also curious about this. Seems to be an issue in GoogleComponent.js. I get a 403 forbidden error. My API key is active and valid. The line that the error points to is this --> var _fire = fetch(this.state.proxyUrl + 'https://maps.googleapis.com/maps/api/place/autocomplete/json?input=' + param + '&' + _lang + _co + '&key=' + this.props.apiKey);

Was this line updated recently? I had a project that I was using react-google-location about 2 years ago but began to revamp it and having an issue with the auto-complete feature.

**Comment by SilverioReyes** [comment]
Did some investigating such as all developers and engineers love to do and completed the challenge :). Anyway, how long will the temporary access to the demo server last?

**Comment by AlgoAbhi** [comment]
@SilverioReyes DId you find any solution? I am facing the same problem.
